cluster/gce/config-default.sh

```diff
@@ -35,11 +35,18 @@ PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}
 
 
-OS_DISTRIBUTION=${KUBE_OS_DISTRIBUTION:-debian}
-MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-container-vm-v20160321}
+# Today the cluster startup scripts asssume the master and the nodes use the
+# same OS distro, thus same set of initialization instructions (e.g., metadata,
+# startup scripts, etc.). The current workaround is the hack in util.sh that
+# reloads <os_distro>/helper.sh in the gap between when the master is created
+# and when the nodes are created.
+# TODO(#26183): Provide a way to differentiate master OS distro and node OS
+# distro.
+OS_DISTRIBUTION=${KUBE_OS_DISTRIBUTION:-gci}
+MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-}
```

Comment:
Can you add a comment that for gci leaving this blank will auto-select an appropriate image?